db/schema.rb

```diff
@@ -47,6 +47,7 @@
     t.integer "zip"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.boolean "active", default: true
```

Comment:
this is a nitpick but I'd make this `active?` just to be consistent with the items table